### PR TITLE
Add energy support for a template temporal model

### PR DESCRIPTION
**Pull request by AtreyeeS**
As we previously discussed, this enables to 
1. read the cat-sdc files. I have put the read method in utils now, because the file format is a bit annoying and might change in the future (we should encourage a "map" based format)
2. use adapt the evaluation for such model
3. enable plotting such models

It might be necessary to add some documentation in the future, maybe as a separate tutorial showing a full simulation with such models

### CODE REVIEW

File: gammapy/modeling/models/utils.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        energy_axis = MapAxis.from_nodes(
+            nodes=energy_hdu.data,
+            unit=energy_hdu.header["TUNIT1"],
+            name="energy",
+            interp="log",
+        )
+        time_hdu = hdul["TIMES"]
+        time_header = time_hdu.header
+        time_header.setdefault("MJDREFI", 0.5)
+        time_header.setdefault("MJDREFF", 55555)
```
Comment (fabiopintore) [inline gammapy/modeling/models/utils.py:23]: Should this be the opposite? I think that MJDREFI is integer part while MJDREFF is the float one.

**Review by fabiopintore — COMMENTED** [review]
Thanks @AtreyeeS . This is very useful and I don't have any major comment.

File: gammapy/modeling/models/utils.py (hunk 1 above)
Comment (AtreyeeS) [reply gammapy/modeling/models/utils.py:23]: Indeed, my bad! Thanks @fabiopintore

**Comment by registerrier** [comment]
Closing before a new PR is opened.

**Pull request closed without merging**